website/qmds/richards_growth_curve.qmd

```diff
@@ -267,39 +272,27 @@ $$L_{\infty R} ~=~ L_1^b \;+\; \frac{L_2^b\,-\,L_1^b}{1-e^{-k(A_2-A_1)}}$$
 We proceed by replacing $L_{\infty R}$ with its definition and then
 gradually simplify the equation:
 
-$$
-L_t ~=~ \bigg[\,L_1^b \;+\; \frac{L_2^b-L_1^b}{1-e^{-k(A_2-A_1)}} \;+\;
+\begin{eqnarray*}
+L_t &=& \bigg[\,L_1^b \;+\; \frac{L_2^b-L_1^b}{1-e^{-k(A_2-A_1)}} \;+\;
          \left(L_1^b\,-\,\Big[L_1^b \,+\,
          \frac{L_2^b-L_1^b}{1-e^{-k(A_2-A_1)}}\Big]\right)
```

Comment:
Could you add \\ after each equation here? Otherwise the quarto website rendering makes it all one line.